src/main/java/com/synopsys/integration/alert/channel/slack/descriptor/SlackDistributionDescriptorActionApi.java

```diff
@@ -64,7 +64,7 @@ public void validateConfig(final Config restModel, final Map<String, String> fie
     }
 
     @Override
-    public void testConfig(final Config restModel) throws IntegrationException {
+    public void testConfig(final Config restModel, final String ignoredDestination) throws IntegrationException {
```

Comment:
Since we have ignoredDestination in several places, would it make sense that those classes have a different descriptor? Or should the classes that use the destination have a different descriptor?

Reply:
Yes, I discussed this with @psantos1113 last week, but we did not come up with a solution.

Reply:
Since it is specific to a channel should we have the ChannelDescriptor class define the abstract method or DescriptorActionApi create a sub-class that is ChannelDescriptorActionApi and the Global and Distribution Action API classes extend that rather than just DescriptorActionApi?